### DISCUSSION TITLE
how can i pass this validation message to outer

### DISCUSSION THREAD

**Discussion by zrwsmd** [eclipse-langium/langium#1423] [Q&A]
thanks a lot ,it truly works,but how can i pass this validation message to outer ,i mean  when vscode extension start,and open below file ,my custom validation message  'program name should not start with a digital' hint can show in the file
![image](https://github.com/eclipse-langium/langium/assets/26995040/e8203295-b55a-4fa3-a464-fb9cd1324077)

_Originally posted by @zrwsmd in https://github.com/eclipse-langium/langium/discussions/1421#discussioncomment-8838355_

**Comment by msujew** [comment]
Hey @zrwsmd,

no idea what's wrong with your vscode extension. It looks like you started off with a GLSP base extension and added Langium in there. Maybe you should start with a Langium project (from the yeoman generator) and add GLSP on top of it. With that, the validator should work as expected.

**Reply by zrwsmd** [reply]
my requirement is just validate st language ,so i am not considering import GLSP related struct,so without GLSP,i cannot exposed to outer porperly?i thought only langium+vscode extension should  handle this requirement

**Reply by msujew** [reply]
I don't really follow. I'm recommending to create a vscode extension from our langium yeoman generator and use that as a basis to build your own language.

I'm not really familiar with GLSP so I don't have any recommendations for that.
